### PR TITLE
Fixes #395, infobox links now give no errors in result count or pagination bar

### LINKED ISSUE #395: Clicking on links in Infobox, gives no result count or proper pagination bar
image
image
But when i type in Mumbai as a fresh query
image
image

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes #395, infobox links now gives a proper pagination bar, and result count
image
image
image
[Demo link
](https://marauderer97.github.io/susper.com/)

### CODE REVIEW

**Comment by Marauderer97** [comment]
@harshit98 Done.

File: src/app/infobox/infobox.component.ts [hunk 1]
```diff
@@ -47,7 +47,18 @@ export class InfoboxComponent implements OnInit {
       }
     });
   }
-
+  getQuery(q) {
```
Comment (nikhilrayaprolu) [inline src/app/infobox/infobox.component.ts]: can you please explain me the root cause of this issue, due to which variable we are facing this issue?

Comment (Marauderer97) [inline src/app/infobox/infobox.component.ts]: The issue was because the query parameters had only query and not the other fields.

**Comment by nikhilrayaprolu** [comment]
closing this pull, as pull request #405 solves all issues #388 , #389 , #404 #395 in one go.

**Pull request closed without merging**